### DISCUSSION TITLE
Unable to add Device Handler in IDE

### DISCUSSION THREAD

**Discussion by SteDowney** [uncdubdiver/smartthings-govee-led-lights#17] [Q&A]
I no longer have the option to add a device handler in the ide and it also looks like Smartthings are removing their support for Groovey.
(https://community.smartthings.com/t/the-end-of-groovy-has-arrived/246280)

Do you know it there are any workarounds to add Govee LED lights to Smartthings?

**Comment by uncdubdiver** [comment]
I have a page that I made that’s web based, not SmartThings. If you want to give that one a try for a bit you’re more than welcome to, however I don’t have anything built into ST at this time for it since I no longer have a ST hub.

 

Dean Berman
DB Development
 ***@***.***> ***@***.***
(O) 704.837.0284
(C) 704.577.1236

         

 

From: SteDowney ***@***.***> 
Sent: Monday, January 16, 2023 9:06 AM
To: uncdubdiver/smartthings-govee-led-lights ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [uncdubdiver/smartthings-govee-led-lights] Unable to add Device Handler in IDE (Discussion #17)

 

I no longer have the option to add a device handler in the ide and it also looks like Smartthings are removing their support for Groovey.
(https://community.smartthings.com/t/the-end-of-groovy-has-arrived/246280)

Do you know it there are any workarounds to add Govee LED lights to Smartthings?

**Reply by tazntampa** [reply]
Hi Dean, I'm trying to integrate my new Govee H605C and 2 H6072 into Smartthings. Seems we can no longer add Device Handler in the account.smartthing.com url. Have you found a new workaround for this or have any suggestions? Thanks in advance - bit of a Novice user here :-)

**Comment by uncdubdiver** [comment]
Unfortunately I no longer have a ST hub so I'm unable to continue any
further development on it since they replaced their integration with a new
form/language.

Sorry :(


Thank You.



*Dean Berman*
DB Development
***@***.***
(O) 704.837.0284
(C) 704.577.1236
